python/README.md

```diff
@@ -4,6 +4,8 @@ ittapi is a Python binding to Intel Instrumentation and Tracing Technology (ITT)
 to mark up the Python code for further performance analysis using performance analyzers from Intel like Intel VTune
 or others.
 
+NOTE: For compatibility and smooth transition with the old version of ITT API(https://github.com/oleksandr-pavlyk/itt-python/tree/master), please replace `import itt` with `import ittapi.compat as itt`. For more details about `ittapi.compat`, visit [ittapi.compat page](./ittapi/compat/README.md).
```

Comment:
```suggestion
NOTE: For compatibility and smooth migration with earlier versions of Python bindings [itt-python](https://github.com/oleksandr-pavlyk/itt-python), please replace `import itt` with `import ittapi.compat as itt`. For more details about `ittapi.compat`, visit [ittapi.compat page](./ittapi/compat/README.md).
```

Reply:
Have made the changes. Please have a look.

Reply:
please hyperlink the early python bindings to the repository, no need to put that URL in the readme text

Reply:
Done.